modules/framework/galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/FrameworkRuns.java

```diff
@@ -618,6 +623,17 @@ private void setRunRequestRequestorDefault(SubmitRunRequest runRequest) throws F
         }
     }
 
+    private void setRunRequestUserDefault(SubmitRunRequest runRequest) throws FrameworkException {
+        String user = AbstractManager.nulled(runRequest.getUser());
+        if (user == null) {
+            user = AbstractManager.nulled(cps.getProperty("run", "user"));
```

Comment:
I see this comes from the `setRunRequestRequestorDefault` method above, but this seems to mean that the user can be set using a CPS property if no user was passed in - is this needed?

I'm not sure why the `setRunRequestRequestorDefault` method has that in either!